python/tvm/relay/frontend/onnx.py

```diff
@@ -2239,7 +2239,8 @@ def _fix_outputs(self, op_name, outputs):
             outputs = outputs[:-1]
         return outputs
 
-def from_onnx(model,
+def from_onnx(model_path=None,
+              external_data_files_path=None,
```

Comment:
Put `external_data_files_path` further down (ideally last) in the argument list. All the importers have the first two arguments as model, shape_dict and it'd be nice to keep this consistent.